src/frontend/src/test-e2e/util.ts

```diff
@@ -23,17 +29,27 @@ export async function runInBrowser(
   // parse run configuration from environment variables
   const runConfig = parseRunConfiguration();
 
+  const chromeOptions: ChromeOptions = {
+    args: [
+      "--ignore-certificate-errors", // allow self-signed certificates
+      "--disable-gpu",
+    ],
+  };
+
+  if (runConfig.screenConfiguration.screenType === "mobile") {
+    chromeOptions.mobileEmulation = {
+      deviceMetrics: runConfig.screenConfiguration.deviceMetrics,
+    };
+  } else {
+    chromeOptions.args?.push(
+      `--window-size=${runConfig.screenConfiguration.windowSize}`
+    );
+  }
+
   const browser = await remote({
     capabilities: {
       browserName: "chrome",
-      "goog:chromeOptions": {
```

Comment:
why did this move? also do we now run with mobile emulation? probably worth mentioning in the PR description

Reply:
This moved in order to more easily set different options depending on the run configuration. The PR description already mentions that mobile emulation is turned on.